src/components/Users/UserSummary.tsx

```diff
@@ -172,17 +172,17 @@ export default function UserSummaryTab({
               </div>
             </div>
             <div className="w-3/4">
-              <ButtonV2
+              <AuthorizedButton
```

Comment:
Use `AuthorizedChild` instead or refactor to write the permissions logic to a `permissions.ts` (the newer way we started following recently and use that fn. here along with `useAuthUser`